modules/learn/pages/security/on-the-wire-security.adoc

```diff
@@ -116,7 +116,7 @@ The following limitations apply to cipher-suite configuration.
 
 As indicated above, TLS 1.3 cipher-suites can by used by all services; and by the Cluster Manager, XDCR, and Views.
 
-However, only _Cluster Manager_ and the _Data_ and _Analytics_ services support the custom-configuration of TLS 1.3 ciphers &#8212; _custom-configuration_ of ciphers means the selection of a subset of available ciphers for a custom cipher-suite list, and the specifying of the selected ciphers in the subset in any order judged appropriate.
+However, only the _Cluster Manager_ and the _Data_ and _Analytics_ services support the custom-configuration of TLS 1.3 ciphers &#8212; _custom-configuration_ of ciphers means the selection of a subset of available ciphers for a custom cipher-suite list, and the specifying of the selected ciphers in the subset in any order judged appropriate.
```

Comment:
I don't mean to nitpick at all but the change I was hoping to see was to use the form:
`only Thing1, Thing2, and FinalThing` ... instead of `only Thing1 and Thing2 and FinalThing`.. See how we have the multiple and's ? Generally we should just use `and` for the final item. The `the` isn't as important, though I guess is also fine.

Reply:
Maybe something like:
```
However, only the _Cluster Manager_, _Data_ and _Analytics_ services support the custom-configuration of TLS 1.3 ciphers ...
```

Reply:
You are not nit picking and I don't mind the discussion at all. But I do disagree.

I  understand (I think) what you are saying and its motivation. However, the use of 'and' is, I believe, required; in order to ensure that 'services' applies only to Thing2 and FinalThing, but not to Thing1. It seems to me that what you are proposing would be interpreted as  "However, only the Cluster Manager service, the Data service, and the Analytics service support the custom-configuration ...etc." This is incorrect, because the Cluster Manager is not a service. I recognise that the absence of a comma after "Data" might mitigate the misinterpretation, but only, I think, to leave the sentence ambiguous.

Regarding the use of 'and' in lists: I agree it is simpler to reserve it for separating the last two items; but all the same, it is very typically used to separate elements within an item. So: "The options for printing include color and black and white." That's two options, not three. 

A rewording that avoids such problems might be: "However, only the Cluster Manager, the Data Service, and the Analytics Service support ... etc". That removes ambiguity, and uses 'and' just once; at the expense of a more formal and necessarily extended naming of the services.

At any rate, I have rewritten the sentence according to the wording you proposed. Let me know if you can approve. Thanks again.

Reply:
Wait so with this explanation maybe we really should do it the way it was before you made this change. I guess I didn't know we didn't consider cluster manager a service.. So @tonyjhillman I will gladly approve either. So if you feel strongly about it feel free to change it and i'll re-approve that as well.